scripting/include/srccoop/globals.inc

```diff
@@ -20,6 +20,7 @@ ConVar g_pConVarNextStuck;
 ConVar mp_friendlyfire;
 ConVar mp_flashlight;
 ConVar sv_always_run;
+ConVar mp_teamplay;
```

Comment:
CBM_MP_GameRules.IsTeamplay() should be accurate during whole map, unlike the cvar which can be changed any time.

Reply:
Ah, that would make it more accurate.